modules/layers/src/geojson-layer/sub-layer-map.js

```diff
@@ -10,6 +10,7 @@ export const POINT_LAYER = {
     props: {
       filled: 'filled',
       stroked: 'stroked',
+      pointBillboard: 'billboard',
```

Comment:
Can we leave this outside of this PR? None of the other sublayers (icon, text, path) are exposing their `billboard` prop. It can be set using `_subLayerProps`